### ISSUE TITLE
Suffix

### ISSUE THREAD

**Issue by LVS04** [WhiteKarma/NalandaERP#13]
Suffix is useless. Should be removed.

**Comment by WhiteKarma** [comment]
OK, thank you, ( the use of the field is for the cases of same name between for example father and son and to be able to distinguish as JR - Junior or SR - Senior ) for example. We can discuss this, yes, since the cases are not so common. 

Thank you!!!

**Comment by LVS04** [comment]
Yes, for sure, we are expecting a lot of Junior and Senior…. :):)

 

De : WhiteKarma [mailto:notifications@github.com] 
Envoyé : mardi 27 septembre 2016 12:11
À : WhiteKarma/NalandaERP <NalandaERP@noreply.github.com>
Cc : LVS04 <lvs04@yahoo.fr>; Author <author@noreply.github.com>
Objet : Re: [WhiteKarma/NalandaERP] Suffix (#13)

 

OK, thank you, ( the use of the field is for the cases of same name between for example father and son and to be able to distinguish as JR - Junior or SR - Senior ) for example. We can discuss this, yes, since the cases are not so common. 

Thank you!!!

**Comment by WhiteKarma** [comment]
HAHAHAHAHAHAHAHAHAH.

I agreee, let's ear some more opinions.

Thank you!!!
